src/panels/lovelace/components/hui-warning.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+
+export class HuiWarning extends LitElement {
+  @property() public warning?: string;
+  @property() public entity?: string;
+
+  public getCardSize(): number {
+    return 1;
+  }
+
+  protected render(): TemplateResult | void {
+    return html`
+      ${this.warning || "Entity not available"}: ${this.entity || ""}
```

Comment:
I think that we should just render `<slot></slot>` and have the warning be passed in: `<hui-warning>${this.hass.localize('ui.lovelace.warning.entity_not_found', 'entity', this.entity}</hui-warning>`

Reply:
You'll have to add the warning string to Lovelace in `translations/en.json` and run `gulp build-translations` to get latest translations build (or restart `script/develop` (slow))